Medicina/CDB/alma/BACKENDS/PyCalmux/PyCalmux.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='ISO-8859-1'?>
+<!--
+     Andrea Orlati, andrea.orlati@inaf.it
+-->
+<PyCalmux
+    xmlns="urn:schemas-cosylab-com:PyCalmux:1.0" 
+    xmlns:baci="urn:schemas-cosylab-com:BACI:1.0"
+    xmlns:cdb="urn:schemas-cosylab-com:CDB:1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    IP="192.168.51.202"
```

Comment:
The IP and port for testing CDB could also be set to the simulator parameters:
127.0.0.1
12500
Your choice